unlock-js/src/__tests__/helpers/nockHelper.js

```diff
@@ -2,14 +2,13 @@
 import nock from 'nock'
 
 export class NockHelper {
-  constructor(endpoint, debug = false, isEthers = false) {
+  constructor(endpoint, debug = false) {
     this.nockScope = nock(endpoint, { encodedQueryParams: true })
 
     this.anyRequestSetUp = false
     this.debug = debug
-    this._rpcRequestId = isEthers ? 42 : 0
+    this._rpcRequestId = 42
```

Comment:
Please let's put a comment here so that this is not very confusing when ethers all of a sudden decides to implement something saner than `42` ;p

Reply:
Ideally maybe even a link to the place where ricmoo discusses this?

Reply:
good idea, just pushed this